src/test/java/io/syndesis/qe/rest/tests/AbstractSyndesisRestTest.java

```diff
@@ -17,20 +24,42 @@
  */
 public abstract class AbstractSyndesisRestTest {
 
-	protected static final String syndesisToken = SyndesisRestConstants.SYNDESIS_TOKEN;
-	protected static final String syndesisURL = SyndesisRestConstants.SYNDESIS_URL;
-	protected static final String openshiftToken = SyndesisRestConstants.OPENSHIFT_TOKEN;
-	protected static final String openshiftUrl = SyndesisRestConstants.OPENSHIFT_URL;
+	private static Account syndesisAccount;
+	private static Account openshiftAccount;
+	private static AccountsDirectory accountsDirectory;
+	private DefaultUserUtil defUser;
+	protected static String syndesisToken;
+	protected static String syndesisURL;
+
+	protected static String openshiftToken;
+	protected static String openshiftUrl;
 	protected static OpenShiftClient openshiftClient;
 
 	public AbstractSyndesisRestTest() {
+
+		accountsDirectory = new AccountsDirectory(Paths.get(SyndesisRestConstants.ACCOUNT_CONFIG_PATH));
+
+		syndesisAccount = accountsDirectory.getAccount("syndesis").get();
+		openshiftAccount = accountsDirectory.getAccount("openshift").get();
+
+		syndesisURL = syndesisAccount.getProperty("instanceUrl");
+		openshiftUrl = openshiftAccount.getProperty("instanceUrl");
+		openshiftToken = openshiftAccount.getProperty("openshiftToken");
 		RestAssured.baseURI = syndesisURL;
 		RestAssured.basePath = SyndesisRestConstants.API_PATH;
 
 		Config config = new ConfigBuilder()
-				.withOauthToken(openshiftToken)
+				.withUsername(syndesisAccount.getProperty("login"))
+				.withPassword(syndesisAccount.getProperty("password"))
```

Comment:
Are you sure that it work with username+password and omitting token?

Reply:
np, tried it. Made this change, because we need the uname/pass specified for case when the OS user is not yet created (happens when we use syndesis on minishift), so the token would be redundant.